graph/quadstore.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 type BatchQuadStore interface {
-	ValuesOf(ctx context.Context, vals []Value) ([]quad.Value, error)
+	ValuesOf(ctx context.Context, vals []Value) ([]quad.Value, error) // TODO: Should be called NamesOf()
+	NamesOf(ctx context.Context, nodes []quad.Value) ([]Value, error) // TODO: Should be called ValuesOf()
```

Comment:
```suggestion
	RefsOf(ctx context.Context, nodes []quad.Value) ([]Value, error) // TODO: Should be called ValuesOf()
```
And the `ValuesOf` above is fine. This is a legacy from an old version of Cayley. `NameOf` and `ValueOf` will be removed in v0.8.